### DISCUSSION TITLE
truenas

### DISCUSSION THREAD

**Discussion by aelsharawi** [tteck/Proxmox#2674] [Help Needed] [answered]
any reason why no truenas
and what is the best way to have it?

**Comment by tteck** [comment, marked as answer]
https://www.truenas.com/

**Reply by aelsharawi** [reply]
so what do you advise Proxmox with Truenas core

or trueNas scale and forget about Proxmox?

**Reply by tteck** [reply]
I advise choosing whatever suits your needs the best.
